### ISSUE TITLE
Reduce fastText models size

### ISSUE THREAD

**Issue by antter** [aicoe-aiops/github-labeler#83]
**Is your feature request related to a problem? Please describe.**
fastText has proven itself to be quite useful in predicting github labels, but the models are way too big, especially for the pre-trained version. It is not convenient to save multiple models and load them in the app. Some care has already been taken to reduce model size, such as reducing vocabulary and vector size in the src/data/build_w2v_vocab notebook. More can still be done to significantly reduce the size.

**Describe the solution you'd like**

Two main ideas are to quantize the model, which fastText allows easily in their python package. Another option is to throw out even more vocabulary (e.g. if a word doesn't appear once in our target issues), and reduce vector size yet again. We could have a finetuned vector model that is useful for anyone, and smaller, specific ones per user.
